pkg/node/host_endpoint.go

```diff
@@ -3,22 +3,31 @@
 
 package node
 
+import (
+	"github.com/cilium/cilium/pkg/lock"
+)
+
 const (
 	templateHostEndpointID = uint64(0xffff)
 )
 
 var (
 	labels     map[string]string
+	labelsMu   lock.Mutex
 	endpointID = templateHostEndpointID
 )
 
 // GetLabels returns the labels of this node.
 func GetLabels() map[string]string {
+	labelsMu.Lock()
+	defer labelsMu.Unlock()
 	return labels
 }
 
 // SetLabels sets the labels of this node.
 func SetLabels(l map[string]string) {
+	labelsMu.Lock()
+	defer labelsMu.Unlock()
 	labels = l
 }
 
```

Comment:
While you are at it, can you fix the comment on line 39? :pray:

Reply:
Sure! Nice catch!